### ISSUE TITLE
Feature Request: Safari Browser Support for Regular / Pro

### ISSUE THREAD

**Issue by badcat** [ThemeFuse/Brizy#579]
I cannot believe you would remove support for Safari browser. It's the default on macOS and what will become of support on iPad?  I hope this is just for version 0.0.4 Pro and not a long term thing. iOS support was one of the things that seemed to set Brizy apart from other page builders, and now it gone?

Please reconsider and return support for Safari and iOS Safari.  thank you.

![unreal](https://user-images.githubusercontent.com/26096/45363368-5976f680-b5a5-11e8-8b0e-322e8d55d3c0.png)

**Comment by willhousecreative** [comment]
I agree with @badcat. IMHO, requiring a specific web browser is short-sighted.

**Comment by site42** [comment]
I very much agree with @badcat. It should be browser independent.

**Comment by badcat** [comment]
Two quick follow ups on this.

1) I'm not the biggest Edge user or supporter, however I feel the same way - Edge is the native browser for Windows and it seems like this too should be supported.

2) Safari has long allowed masquerading as a different User Agents (via their Preferences > Advanced - Develop menu). Switching to Firefox or Chrome in this way appears to allow Brizy to work without issue.

So I suppose the question here now is - Why doesn't Brizy just support Safari (and Edge) ?  
Was there some technical issue, or is this a support issue?

thanks for listening.

**Comment by site42** [comment]
On iPad it doesn’t work either with Firefox or Chrome for that matter; it says “Sorry, Safari is not.....”??
![b7d97cc6-9d74-4725-a99b-8dbc407b5730](https://user-images.githubusercontent.com/43269112/46905292-c0afff80-cef1-11e8-975b-ee16664ae80b.jpeg)
I guess iPad (IOS) is not (fully) supported or am I misging something 😉 ?

**Comment by badcat** [comment]
I'd guess it's not working on Firefox and Chrome on iPad because both browsers are based on Safari's WebKit engine (not Google's V8 etc). Brizy is likely only (incorrectly) identifying the browser, not the rendering engine.  

Unfortunately, brizy seems crippled for any editing on mobile devices at this point.  Gotta wonder why - The numbers of people using non-computer (desktop, laptop etc) is only increasing every year.  Mobile editing was one of the selling points of brizy for me.  Shame we've lost it.

**Comment by storiwr** [comment]
Adding my vote for this feature. Safari/webkit support is essential. I work a lot on the go and Safari is the only browser that doesn't drain my battery (quite as fast). Plus, I find it's faster and less resource intensive, at least for what I do.
